BGK.Data/Repositories/ClassroomRepository.cs

```diff
@@ -28,5 +28,10 @@ public bool UpdateClassroomInfo(Classroom newInfo)
             _db.SaveChanges();
             return true;
         }
+
+        public List<Classroom> GetClassroom()
+        {
+            return _db.Classroom.ToList();
+        }
```

Comment:
1. please rename this method properly (GetClassrooms instead of GetClassroom). 
2. use GetInstance method instead of "new ApplicationContext()" at the top of this class

Reply:
Вероятно ты имел ввиду метод  Storage.GetStorage();